### PR TITLE
Style Guide: Clarify video instructions for "What's new" and Wistia

### PR DESCRIPTION
**Pull request by rhetoric101**

### CODE REVIEW

File: src/content/docs/style-guide/images/embed-videos.mdx [hunk 1]
```diff
@@ -54,6 +54,13 @@ To upload videos to Wistia:
   />
   ```
 
+<Callout variant="tip">
+  For "What's new" posts that use `.md` files, you can't use the video component. Instead, you'd use an `iframe` tag like this: 
+  ```html
+  <iframe width="560" height="315" src="https://fast.wistia.net/embed/iframe/o8r4tfsa8l" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (paperclypse) [inline src/content/docs/style-guide/images/embed-videos.mdx:60]: Should we replace `o8r4tfsa8l` with a placeholder value?

**Review by paperclypse — DISMISSED** [review]
left a couple comments

File: src/content/docs/style-guide/images/embed-videos.mdx [hunk 2]
```diff
@@ -54,6 +54,13 @@ To upload videos to Wistia:
   />
   ```
 
+<Callout variant="tip">
+  For "What's new" posts that use `.md` files, you can't use the video component. Instead, you'd use an `iframe` tag like this:
```
[the remark this answers is not in the document]
Comment (rhetoric101) [reply src/content/docs/style-guide/images/embed-videos.mdx]: @paperclypse We're not consistent in our we capitalized this feature. I'll switch to What's New here and we can address the larger question later.

File: src/content/docs/style-guide/images/embed-videos.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/style-guide/images/embed-videos.mdx]: I replaced the code with a placeholder.

**Pull request merged**